scss/_variables.scss

```diff
@@ -2311,6 +2311,11 @@ $quantity-selector-btn-padding-x-sm:      add($btn-icon-padding-x-sm, 2px) !defa
 $quantity-selector-icon-width:            .875rem !default;
 $quantity-selector-icon-sm-width:         .625rem !default;
 
+$quantity-selector-disabled-color:        var(--#{$prefix}disabled-color) !default;
+$quantity-selector-disabled-bg:           var(--#{$prefix}body-bg) !default;
+$quantity-selector-button-bg:             var(--#{$prefix}body-bg) !default;
```

Comment:
We have `$quantity-selector-btn-padding-x`, so this one shouldn't be grouped together with those `$quantity-selector-btn-*` vars and be named `*-btn-*` instead of `*-button-*`?